plutus-core/untyped-plutus-core/test/Evaluation/Machines.hs

```diff
@@ -119,19 +119,15 @@ test_budget :: TestTree
 test_budget
     -- Error diffs are very big
     = localOption (SizeCutoff 1000000)
-    . runTestNestedIn ["untyped-plutus-core", "test", "Evaluation", "Machines"]
-    . testNested "Budget"
+    . runTestNested ["untyped-plutus-core", "test", "Evaluation", "Machines", "Budget"]
     $ concat
         [ folder Plc.defaultBuiltinsRuntime bunchOfFibs
         , folder (toBuiltinsRuntime def ()) bunchOfIdNats
         , folder Plc.defaultBuiltinsRuntime bunchOfIfThenElseNats
         ]
   where
     folder runtime =
-        foldPlcFolderContents
-            testNested
-            (\name _ -> pure $ testGroup name [])
-            (\name -> testBudget runtime name . eraseTerm)
+        foldPlcFolderContents testNested mempty (\name -> testBudget runtime name . eraseTerm)
```

Comment:
Being able to write `mempty` instead of that long `\name _ -> pure $ testGroup name [])` is pretty convenient. In general it's quite nice to have an interface that is both monadic and monoidal.